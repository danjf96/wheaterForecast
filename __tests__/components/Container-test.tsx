/**
 * @format
 */

 import 'react-native';
 import React from 'react';
 import Container from '../../src/components/container'
 // Note: test renderer must be required after react-native.
 import { render } from 'react-native-testing-library';
 
 it('renders correctly', async () => {
   const screen = render(<Container />)
   expect(screen.toJSON()).toMatchSnapshot()
 });

 