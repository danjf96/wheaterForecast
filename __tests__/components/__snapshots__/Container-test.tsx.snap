// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<BVLinearGradient
  colors={
    Array [
      4278224639,
      4288499932,
    ]
  }
  endPoint={
    Object {
      "x": 0.5,
      "y": 1,
    }
  }
  locations={null}
  startPoint={
    Object {
      "x": 0.5,
      "y": 0,
    }
  }
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;
